packages/backend-modules/search/lib/inserts/document.js

```diff
@@ -64,53 +69,60 @@ const upsertResolvedMeta = ({ indexName, entities, type, elastic }) => {
             bool: {
               filter: {
                 terms: {
-                  [`meta.${type}`]: getResourceUrls(repoName),
+                  [`meta.${metaType}`]: getResourceUrls(repoName),
                 },
               },
             },
           },
         },
       })
       .catch((e) => {
-        console.log(entity, repoName, type, e.meta?.body?.failures)
+        console.log(entity, repoName, metaType, e.meta?.body?.failures)
         return []
       })
 
     if (result.failures?.length > 0) {
       console.error(entity.repoId, result.failures)
     }
-  })
+
+    results.push(result)
+  }
+
+  return results
 }
 
 const after = async ({ indexName, elastic }) => {
   const dossiers = await findTemplates(elastic, 'dossier')
```

Comment:
side note: dossiers are completely obsolete at this point (in: no one created one in the past 5 years). very much in the own-or-kill pipeline.